code/2.arbeit.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arbeitserfahrung</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+<body>
+    
+    <div class="menu">
+        <menu>
+          <a href="./index.html">Home</a>  
+          <a href="./2.arbeit.html">Arbeitserfahrung</a>
+          <a href="./3.ausbildung.html">Ausbildung</a>
+          <a href="./4.sprachen.html">Sprachen</a>
+          <a href="./5.weiteres.html">Weiteres</a>
+          <a href="./6.IT.html">IT Kentnisse</a>
+          <a href="./7.anhang.html">Anhang</a>
+        </menu>
+    </div>
+    
+ <h1>Arbeitserfahrung</h1>   

```

Comment:
Be consistent with the indentation (2 spaces). "Siblings" should be on the same level of indentation and descendants/children should be indented one level:
```
<body>
  <div class="menu">
    <menu>
      <a href="./index.html">Home</a>  
      <a href="./2.arbeit.html">Arbeitserfahrung</a>
      <a href="./3.ausbildung.html">Ausbildung</a>
      <a href="./4.sprachen.html">Sprachen</a>
      <a href="./5.weiteres.html">Weiteres</a>
      <a href="./6.IT.html">IT Kentnisse</a>
      <a href="./7.anhang.html">Anhang</a>
    </menu>
  </div>
  <h1>Arbeitserfahrung</h1> 
```